cluster/libvirt-coreos/util.sh

```diff
@@ -140,6 +143,18 @@ function initialize-pool {
       render-template "$ROOT/skydns-rc.yaml"  > "$POOL_PATH/kubernetes/addons/skydns-rc.yaml"
   fi
 
+  mkdir -p "$POOL_PATH/etcd"
+  if [[ ! -f "$ROOT/etcd-v2.0.9-linux-amd64.tar.gz" ]]; then
+      wget -P "$ROOT" https://github.com/coreos/etcd/releases/download/v2.0.9/etcd-v2.0.9-linux-amd64.tar.gz
+  fi
+  if [[ "$ROOT/etcd-v2.0.9-linux-amd64.tar.gz" -nt "$POOL_PATH/etcd/etcd" ]]; then
```

Comment:
@dchen1107 I have factorized the version of etcd to deploy.

If we pull etcd as a docker image, it means that we will need to pull it each time the cluster is booted. Here, etcd is downloaded only the first time the cluster is booted.

Even if it is not a requirement of kubernetes, `libvirt-coreos` uses an etcd cluster with one etcd instance on each VM. Pulling it as a docker image would require to pull the image on every VM whereas here, it is downloaded once for all and made available to all VMs via VM drives.